packages/protocol/contracts/bridge/IBridge.sol

```diff
@@ -8,13 +8,13 @@ pragma solidity ^0.8.20;
 
 import { LibBridgeData } from "./libs/LibBridgeData.sol";
 /**
- * Recalling a message interface.
+ * An interface that all recallable message sender shall implement.
  */
 
 interface IRecallableMessageSender {
     function onMessageRecalled(IBridge.Message calldata message)
         external
-        returns (bytes4);
+        payable;
```

Comment:
message.callValue will be send to this contract as well. This is the couterparty of invoke message on the destination chain.